readme.md

```diff
@@ -1,321 +1,31 @@
-
-### Obtaining current session data
-
-Once authentication has been established, the Map or DefineMap provided as the `Map` option on the can-connect Model will have a new `current` property defined.  So, if you passed a `Session` object, `Session.current` will always hold the current session data.  This greatly simplifies the session property in your application ViewModel.  Here's an abbreviated example.
-
-```js
-import Session from 'my-app/models/session';
-
-const AppViewModel = DefineMap.extend({
-  session: {
-    get () {
-      return Session.current;
-    }
-  }
-});
-```
-
-That's it!  The `session` property in the above example will automatically populate when the user authenticates.
-
-### Handling OAuth Logins
-
-The `feathers-session` behavior is preconfigured to listen to `login` messages coming in over the [feathers-authentication-popups](https://github.com/feathersjs/feathers-authentication-popups) `authAgent`.  When any message is received through the `authAgent`, its validity is checked.  If it's a valid JWT token, a Session instance will be created automatically.  This will both populate `Session.current` and dispatch a `created` event on the connected Session Map.
-
-### Session Behavior Example
-
-Here's an example Todo Model that uses the can-connect-feathers service behavior.  Comments indicating key locations have been added.  Also note that you'll need to setup a `User` Model for this example to work.  It should use the `service` behavior.
-
-```js
-import connect from 'can-connect';
-import DefineMap from 'can-define/map/';
-
-import feathersSessionBehavior from 'can-connect-feathers/session';
-import dataParse from 'can-connect/data/parse/';
-import construct from 'can-connect/constructor/';
-import constructStore from 'can-connect/constructor/store/';
-import constructCallbacksOnce from 'can-connect/constructor/callbacks-once/';
-import canMap from 'can-connect/can/map/';
-import canRef from 'can-connect/can/ref/';
-import dataCallbacks from 'can-connect/data/callbacks/';
-
-// Bring in the feathersClient instance.
-import feathersClient from './feathers';
-
-export const Session = DefineMap.extend('Session', {
-  seal: false
-}, {
-  exp: 'any',
-  userId: 'any',
-  user: {
-    Type: User,
-    // Automatically populate the user data when a userId is received.
-    get (lastSetVal, resolve) {
-      if (lastSetVal) {
-        return lastSetVal;
-      }
-      if (this.userId) {
-        User.get({_id: this.userId}).then(resolve);
-      }
-    }
-  }
-});
-
-connect([
-  // Include the feathers session behavior in the behaviors list.
-  feathersSession,
-  dataParse,
-  canMap,
-  canRef,
-  construct,
-  constructStore,
-  constructCallbacksOnce,
-  // Include the realtime behavior.
-  realtime,
-  dataCallbacks
-], {
-  // Pass the feathers client as the `feathersClient` property.
-  feathersClient: feathersClient,
-  idProp: 'exp',
-  // Pass the Session Map
-  Map: Session,
-  name: 'session'
-});
```

Comment:
We should make sure all of these docs & examples are in the docs rendered on canjs.com